utils/hashing-proc-macro/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "hashing-proc-macro"
```

Comment:
We should come up with a better name.

Reply:
I feel like the full name would be `sp-core-hashing-proc-macro`. because it is the proc-macro version of the hasher that are in sp-core::hashing.

Reply:
Sounds like a good direction to me, I will do that soon.